tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1277,3 +1277,109 @@ Events:
 		This event forwards data received over L2CAP channel.
 		Chan_ID is the internal application number that identifies
 		L2CAP channel.
+
+Mesh Node Service (ID 4)
+==================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Init command/response
+		Controller Index:	<controller id>
+		Command parameters:	Static Auth (16 octets)
+					Output Size (1 octet)
```

Comment:
Not sure if I get it. Why cannot use bitmask to specify multiple actions? Is output size parameter necesary? The same question applies to Input size/actions